src/rates/rates.service.ts

```diff
@@ -78,6 +78,9 @@ export class RatesService {
     if (amount === 0) {
       return;
     }
+    if (matter === 'CIC' && raterProfileId === matterTargetId) {
+      throw new BadRequestException('User cannot rate itself');
```

Comment:
Might be better to say "Users cannot rate themselves"

Reply:
Fixed